conf/messages.cy

```diff
@@ -798,7 +798,7 @@ tai.incomes.updated.check.heading.old=Mae’ch incwm trethadwy oddi wrth {0} wed
 tai.incomes.updated.check.heading=Mae’ch diweddariad wedi dod i law.
 tai.incomes.updated.check.heading.pt2=Bydd eich cyfrif yn cael ei ddiweddaru cyn pen
 tai.incomes.updated.check.heading.pt3=48 awr
-tai.incomes.updated.check.button=Dychwelyd i’ch incwm amcangyfrifedig
+tai.incomes.updated.check.button=Dychwelyd i'ch manylion cyflogaeth
```

Comment:
can you use smart quote instead of normal apostrophe?
http://smartquotesforsmartpeople.com/
 apostrophe won't be showing in the actual page if you have a look because it ignores 1 apostrophe